### ISSUE TITLE
`@note` annotation is only parsed if separated by new lines

### ISSUE THREAD

**Issue by meleyal** [realm/jazzy#413]
Our code is commented as follows:

```
/// First line of comment.
/// Second line of comment
/// @note This is a note.
/// Final line of comment.
```

The `@note` is not parsed correctly and is just shown inline with the regular comment text.

The following works, but ideally we'd like to avoid introducing those new lines into an already large code-base:

```
/// First line of comment.
/// Second line of comment
///
/// @note This is a note.
///
/// Final line of comment.
```

**Comment by jpsim** [comment]
I agree, we should make that work. The reason it doesn't today is that we replace `@note` to `- note:` and treat it as a Markdown list to combine our Swift doc comment support with Objective-C's, and Markdown requires there to be space before and after a bulleted list. But we should still fix this for Objective-C.

**Comment by istx25** [comment]
Any progress on this since Dec 2015?

**Comment by jpsim** [comment]
The prognosis is still correct.

**Comment by johnfairh** [comment]
This one tracking issues with newlines and headerdoc-style comments in sourcekitten.
